docs/Maintainer-Guidelines.md

```diff
@@ -105,6 +107,28 @@ is a good opportunity to do it) provided the line itself has some kind
 of modification that is not whitespace in it. But be careful about
 making changes to inline patches—make sure they still apply.
 
+### Adding or updating formulae
+Only one maintainer is necessary to approve and merge the addition of a new or updated formula which passes CI. However, if the formula addition or update is controversial the maintainer who adds it will be expected to fix issues that arise with it in future.
```

Comment:
Nitpick: Perhaps "Any" instead of "Only" here.

Nitpick: "is controversial" ➡️ "proves controversial"

Not Nitpick: "fix issues" ➡️ "address requests and problems"